src/test/java/scenarios/PaymentsAndRefundTest.java

```diff
@@ -266,7 +265,8 @@ public void verifyPaymentByMultipleModeThroughStoreManager(){
         String expectedBalanceDue2 = session.getBillPage().convertToNumberFormat(expBalanceDue2);
         float amtPaid2 = updateAmount1 + updateAmount2;
         String amountPaid2 = session.getBillPage().convertToNumberFormat(amtPaid2);
-        WebdriverWaits.waitForElementVisible(session.getPaymentsPage().totalPaidAmt,5);
+//        WebdriverWaits.waitForElementUntilVisible(session.getPaymentsPage().totalPaidAmt,5);
+        WebdriverWaits.sleep(3000);
```

Comment:
Using WebdriverWaits.waitForElementVisible along with Thread.sleep() introduces redundancy and inefficiency. Can replace Thread.sleep() with dynamic waits, as they are more reliable and optimize execution time. It will improve efficiency as well